plugins/org.eclipse.n4js.xpect.ui/src/org/eclipse/n4js/xpect/ui/common/XpectN4JSES5TranspilerHelper.java

```diff
@@ -125,7 +124,7 @@ public String doCompileAndExecute(final XtextResource resource,
 
 		loadXpectConfiguration(init, fileSetupContext);
 
-		File artificialRoot = Files.createTempDirectory("n4jsXpect").toFile();
+		Path artificialRoot = Files.createTempDirectory("n4jsXpectOutputTest");
 
 		RunConfiguration runConfig;
 		// if Xpect configured workspace is null, this has been triggered directly in the IDE
```

Comment:
Is the comment in the next line (line 132) still necessary?

Reply:
No idea.
Removed.